Counterexamples/CliffordAlgebra_not_injective.lean

```diff
@@ -28,6 +29,9 @@ The outline is that we define:
 and discover that $αβγ ≠ 0$ as an element of $K$, but $αβγ = 0$ as an element of $𝒞l(Q)$.
 
 Some Zulip discussion at https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/.F0.9D.94.BD.E2.82.82.5B.CE.B1.2C.20.CE.B2.2C.20.CE.B3.5D.20.2F.20.28.CE.B1.C2.B2.2C.20.CE.B2.C2.B2.2C.20.CE.B3.C2.B2.29/near/222716333.
+
```

Comment:
in the definition of $Q$ above, it should be $Q(\overline{ax+by+cz})=a^2+b^2+c^2$.

Reply:
Fixed, well spotted!